config/module.config.php

```diff
@@ -25,6 +25,7 @@
                 'SlmQueue\Strategy\MaxRunsStrategy' => array('max_runs' => 100000),
                 'SlmQueue\Strategy\MaxMemoryStrategy' => array('max_memory' => 100 * 1024 * 1024),
                 'SlmQueue\Strategy\InterruptStrategy',
+                'SlmQueue\Strategy\MaxPollingFrequencyStrategy' => array('max_frequency' => 0)
```

Comment:
Yes I agree, this should not be enabled by default, and would be "kind-of" BC as it would slow down considerably for people that can afford the cost of constantly polling SQS.